core/src/banking_stage/transaction_scheduler/transaction_priority_id.rs

```diff
@@ -20,7 +19,9 @@ impl TransactionPriorityId {
 
 impl Ord for TransactionPriorityId {
     fn cmp(&self, other: &Self) -> std::cmp::Ordering {
-        self.priority.cmp(&other.priority)
+        self.priority
+            .cmp(&other.priority)
+            .then_with(|| self.id.cmp(&other.id))
```

Comment:
nit: can be write as `(self.priority, self.id).cmp((&(other.priority, self.id))`

even more, i think we just can now `#[derive(Ord)]`. :)

Reply:
yeah - those would have been equivalent.
Think deriving is best option!

Reply:
cd91f84080